docs/GraFx-Studio/guides/anchoring/index.md

```diff
@@ -0,0 +1,179 @@
+# Anchoring
+
+## Concept
+
+See the [Anchoring Concept page](/GraFx-Studio/concepts/anchoring/) for a conceptual explanation.
+
+## Take your time to explore
+
+Anchoring can be explored by creating a blank document with at least two frames and following along with the guide below.
+
+Select a frame to access anchoring properties in the properties panel.
+
+## The Default
+
+When a new frame (text, image, etc.) is created, anchoring is set to Relative by default.
+
+The default Target is set to Page.
+
+![screenshot-full](anchor_guide_02.png)
+
+## Properties Menu
+
+Select a frame, and you'll see the **Anchoring** section in the properties panel. Additional options may appear based on your selections.
+
+![screenshot-full](anchor_guide_01.png)
+
+## Types of Anchors
+
+Select a frame, then adjust the Horizontal and/or Vertical anchor.
+
+![screenshot-full](anchor_guide_05.gif)
+
+### Relative (%)
+
+The image frame below is anchored to the page by default, with Left and Right set to 5%, Top at 14%, and Bottom at 38%.
+
+![screenshot-full](anchor_guide_03.png)
+
+If the canvas size changes (e.g., from 1080 px to 1200 px), the frame's relative position remains the same at 5% from the left and right. However, the actual pixel distance will change:
+
+- 5% of 1080 = 54 px
+- 5% of 1200 = 60 px
+
+!!! Info "Precision"
+    The anchors on screen only show full percentages. Behind the screens GraFx Studio calculates with **4** decimals precision.
```

Comment:
I asked the team to also show up to 2 decimal places for relative offsets. The same as we do with fixed offsets. This means we would show 54.14%.

Also not sure about the internal precision. I think it's more than 4 decimals.